### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by emyrue**
- Used webpack to bundle JavaScript
- Used ES6 modules 
- Dynamically created list of things to do

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "todo",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
```
Comment (GzimAsani) [inline package.json:8]: - [ ] Your webpack is being compiled with errors. Please fix this:
Screenshot from 2022-03-22 22-34-43

File: src/list.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export default class List {
+  constructor() {
+    this.list = [];
```
Comment (GzimAsani) [inline src/list.js:3]: - [ ] The array of  to-do tasks should be an array of objects with each object containing three keys - description [string], completed [bool], index: [number]

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi  @emyrue  
Good job so far! You are almost done :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good job with the gitflow :+1:
- Good PR description :clap: 

## Required Changes ♻️ 
- [ ] _Check the comments below :arrow_double_down: _

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by emyrue** [comment]
If you look at listItem.js, you will see the class that creates the objects (each with three keys) that go into the array that the class in list.js creates.

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job.  :trophy: There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**